neurons/validator.py

```diff
@@ -681,12 +681,14 @@ async def check_videos_and_calculate_rewards_youtube(
             for idx in range(len(description_relevance_scores)):
                 unique_token_count = len(set(TOKENIZER_V2(metadata[idx].description).nonzero()))
                 if unique_token_count <= MIN_LENGTH_BOOST_TOKEN_COUNT:
+                    bt.logging.debug(f"Very few tokens, applying {1.0 - DESCRIPTION_LENGTH_WEIGHT} penalty.")
```

Comment:
I think the comment here should just be `... applying {DESCRIPTION_LENGTH_WEIGHT}`, since 1 - weight is the weight remaining (i.e. 70%)

Reply:
sounds good!